manifest/models.py

```diff
@@ -43,9 +50,14 @@ def __init__(self, timestamp: str = None):
         self.msg_validation: str = ManifestStatus.NOT_SET
 
 
-# TODO - Resource data model
-class ManifestResource(object):
+class ManifestResource:
+    """Data model for manifest resource"""
     def __init__(self, timestamp: str = None):
+        """Constructor for the Manifest file root content
```

Comment:
We need something similar, documenting a data model needs information more on the attributes and the purpose and meaning of the data object, than its methods, as they usually are constructors of some kind